roles/ifw/README.md

```diff
@@ -0,0 +1,259 @@
+# Role netways.icinga.ifw
+
+This role manages the installation/removal of [Icinga for Windows](https://icinga.com/docs/icinga-for-windows/latest/doc/000-Introduction/) components.  
+It first installs the [Icinga PowerShell Framework](https://github.com/Icinga/icinga-powershell-framework) in case it is not present yet and then goes on to use the framework's commands to manage other components.
+
+Tasks it can do:
+
+* Install the Icinga PowerShell Framework
+* Manage repositories (no locally synced repositories yet)
+* Configure the Icinga 2 Agent
+* Create a valid Icinga 2 certificate
```

Comment:
Should we mention here also explicitly the management of the Icinga Background Daemon?